deebotozmo/commands.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, speed: str):
         })
 
 
+class SetScheduleOnce(Command):
+
+    def __init__(self, hour: str, minute: str ):
```

Comment:
```suggestion
    def __init__(self, hour: int, minute: int ):
```

Reply:
ScheduleBase sounds good. But i don't know if i find time to add more. I'm back to int. I was not familar with click.
I had to change just type=click.STRING to type=click.INT.

Reply:
No problem. Don‘t hurry, take your time. I also develop this library in my free time so it will take time 😊

Please be aware that the cli is a small usecase, the lib is used also in other projects. Use the correct type where possible